build-system/babel-config/minified-config.js

```diff
@@ -74,10 +74,31 @@ function getMinifiedConfig() {
     sourceMaps: 'inline',
     presets: [presetEnv],
     retainLines: true,
+    // Babel assumptions enables transformation optimizations.
+    // Full details at https://babeljs.io/docs/en/assumptions.
     assumptions: {
+      arrayLikeIsIterable: true,
+      constantReexports: true,
       constantSuper: true,
+      enumerableModuleMeta: true,
+      // TODO: determine if ignoreFunctionLength can be enabled
+      // https://babeljs.io/docs/en/assumptions#ignorefunctionlength
+      ignoreFunctionLength: false,
+      ignoreToPrimitiveHint: true,
+      iterableIsArray: false,
```

Comment:
Correct, with how many `Set`s we now have, it's very likely we don't always iterate arrays-likes.